ginga/AstroImage.py

```diff
@@ -221,27 +184,19 @@ def set_io(self, io):
     def get_data_size(self):
         return self.get_size()
 
-    def get_header(self, create=True, include_primary_header=None):
+    def get_header(self, create=True, include_primary_header=False):
```

Comment:
Is this a behavior change, so would warrant mention in change log too but no need to undo the change.